test/service/test_portal_service.py

```diff
@@ -84,48 +85,47 @@ def plugin_db(self) -> JSONs:
 
                 }
             ]
-        }
-    ]
+        }]
 
-    demultiplex_db = [
-        {
+    @property
+    def demultiplex_db(self) -> JSONs:
```

Comment:
Why was this converted from an attribute to a property? If for consistency, then please revert. This is less efficient and more verbose that the current implementation. If to enable mutability of the returned value, the redeclare as MutableJSON. If for other reasons, please explain those here.